src/System.Security.Permissions/src/System/Security/Permissions/EnvironmentPermissionAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace System.Security.Permissions
+{
+    [System.AttributeUsageAttribute((System.AttributeTargets)(109), AllowMultiple = true, Inherited = false)]
+    public sealed partial class EnvironmentPermissionAttribute : System.Security.Permissions.CodeAccessSecurityAttribute
+    {
+        public EnvironmentPermissionAttribute(System.Security.Permissions.SecurityAction action) : base(default(System.Security.Permissions.SecurityAction)) { }
+        public string All { get { return default(string); } set { } }
+        public string Read { get { return default(string); } set { } }
+        public string Write { get { return default(string); } set { } }
```

Comment:
For properties like this, you could just define them like:
```C#
public string Read { get; set; }
```
which will have the compiler do the work of creating a backing field, having the getter return its value and having the setter set it.  At least then a value set will roundtrip through the get.